superset-frontend/src/dashboard/components/PropertiesModal.tsx

```diff
@@ -403,7 +419,6 @@ class PropertiesModal extends React.PureComponent {
                   <StyledJsonEditor
                     showLoadingForImport
                     name="json_metadata"
-                    defaultValue={this.defaultMetadataValue}
```

Comment:
what's the implication of removing this line?

Reply:
this.defaultMetadataValue does not exist is not defined anywhere in this modal @etr2460